client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-source-site/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@automattic/components';
+import { StepContainer } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const SiteMigrationSource: Step = function ( { navigation } ) {
+	const translate = useTranslate();
+
+	const handleSubmit = () => {
+		navigation.submit?.();
+	};
+
+	const stepContent = (
+		<div>
+			<p>Enter the url of the site you're migrating.</p>
```

Comment:
shouldn't this be wrapped in a translate call? or are you purposely avoiding that because it might change down the road? 
I think it's safe to translate anyway, but it's your call

Reply:
From my notes above 😁 

> Since the content of this step is placeholder-only, we're not doing any translation calls yet.

I don't think it's worth translating because the copy and layout are still changing.